src/types/schema.d.ts

```diff
@@ -0,0 +1,162 @@
+type DateTime = string;
+type Varchar500 = string;
+type Varchar1000 = string;
+
+export interface Task {
```

Comment:
The `Task`, `Crawl`, `SiteVisit`, `FlashCookie`, `ProfileCookie`, and `CrawlHistoryEntry` can all be dropped. These are OpenWPM tables that are generated by code outside the extension instrumentation. They won't be needed here.